modules/internal/fixInternalDocs.sh

```diff
@@ -62,7 +62,7 @@ function fixTitle() {
   # Module: ChapelFoo
   local titleLen=${#1}
   local header="$(printf '=%.0s' $(seq 1 $titleLen))"
```

Comment:
Aren't they still being used in `  perl -0777 -i -pe "s/$base\n=+\n/$1\n$header\n/g" $2 `?